nemo/collections/__init__.py

```diff
@@ -11,3 +11,5 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+# __all__ = ["asr", "common", "cv", "nlp", "tts"]
```

Comment:
Revert. This if uncommented requires all Nemo collections to be installed all the time.

Reply:
removed